lib/client.js

```diff
@@ -991,6 +991,26 @@
     this.set(path, gauge, opts, handler);
   });
 
+  HappnClient.prototype.publish = maybePromisify(function(path, data, options, handler) {
```

Comment:
Would this work with older happner-2 servers?

Reply:
It would work, but the data would get returned - as the noDataResponse would not mean anything to them